### ISSUE TITLE
Update Batch link creation for teacher dashboard

### ISSUE THREAD

**Issue by sudo-NithishKarthik** [Vowel-Learning/vam-marketplace-react#32]
<img width="478" alt="Screenshot 2021-04-09 at 9 36 30 AM" src="https://user-images.githubusercontent.com/75248557/114126960-132dd380-9917-11eb-829b-37ee90ccb002.png">

-  Change the 'share' button's name  to 'update'
-  Add days of the batch
-  Change 'Date/Time' field to start and end timings
- Make the schedule table hidden

**Comment by bajaj99prashant** [comment]
For sharing the meeting url is there any API?

**Comment by bajaj99prashant** [comment]
Updated the UI of the batches and removed the time table.

**Comment by sudo-NithishKarthik** [comment]
No..share just updates the meeting url, we will just fetch the data from batches meeting url only
